public/index.html

```diff
@@ -60,6 +60,10 @@ <h1 id="title">Peter Bucy</h1>
 
 				<!-- Intro -->
 					<section id="top" class="one dark cover">
+							<video autoplay muted loop id="myVideo">
+									<source src="images/fishvideo.mp4" type="video/mp4">
+								  </video>

```

Comment:
package.json and package.lock should be outside of public at the root because the packages are installed in the server level. need link to deployed as well as the link to the repo! good work on the breakpoints and the video!
the styling for the video needs some work for the other views besides desktop looks a little fishy on the mobile!😹